python/GafferUI/ColorSwatchPlugValueWidget.py

```diff
@@ -197,14 +197,10 @@ def __buttonClicked( self, button ) :
 				for p, v in self.__initialValues.items() :
 					p.setValue( v )
 
-		# ideally we'd just remove ourselves from our parent immediately, but that would
-		# trigger this bug :
-		#
-		# 	https://bugreports.qt-project.org/browse/QTBUG-26761
-		#
-		# so instead we destroy ourselves on the next idle event.
-
-		GafferUI.EventLoop.addIdleCallback( self.__destroy )
```

Comment:
@johnhaddon I think we don't technically need the `return False` + comment now in `__destroy`?

Reply:
Well spotted - squashed into 850aad4.

Reply:
Thanks @johnhaddon, will merge post CI